apps/frontend/src/components/global/checklist/ChecklistSeverityOverride.vue

```diff
@@ -0,0 +1,68 @@
+<template>
```

Comment:
when the sheet pops up with the textbox to provide override text, have it be autopopulated with the already entered override text if it exists.

Reply:
Done